src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <label>
+        <div class="container">
+          <img
+            src="./images/Search.svg" alt="loop"
+            class="loop"
+          >
+          <input
+            type="search"
+            class="input"
+            data-qa="keypress"
+            placeholder="Try &#34;Los Angeles&#34;"
+          >
```

Comment:
add blank line between same level elements
```suggestion
          <img
            src="./images/Search.svg" alt="loop"
            class="loop"
          >
          
          <input
            type="search"
            class="input"
            data-qa="keypress"
            placeholder="Try &#34;Los Angeles&#34;"
          >
```